common/common-shim/src/test/java/org/pentaho/hadoop/shim/common/format/PentahoParquetInputFormatTest.java

```diff
@@ -21,11 +21,13 @@
  ******************************************************************************/
 package org.pentaho.hadoop.shim.common.format;
 
+import java.nio.file.NoSuchFileException;
 import java.sql.Timestamp;
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 
+import jdk.nashorn.internal.ir.annotations.Ignore;
```

Comment:
Don't you want org.junit.Ignore?
And actually- did you intend to ignore the new test?

Reply:
I ignored, because locally is running with success and fails when running at build server. Today I'm going to try understand why. But I think that the code is safe. Tested and re-tested.

Reply:
Still pretty sure you didn't mean to use nashorn, though.  :)